src/rpm/const.go

```diff
@@ -0,0 +1,124 @@
+package rpm
+
+const (
+	defaultFileUser   = "root"
+	defaultFileGroup  = "root"
+	defaultFileLang   = ""
+	defaultFileLinkTo = ""
+	emptyDigest       = ""
+
+	headerSignatures = 62
+	headerImmutable  = 63
+
+	hashAlgoSHA256 = 8
+
+	// XXX
+	fileFlag = 1 << 4
+	dirFlag  = 0
+
+	rpmTypeNull        = 0
+	rpmTypeChar        = 1
+	rpmTypeInt8        = 2
+	rpmTypeInt16       = 3
+	rpmTypeInt32       = 4
+	rpmTypeInt64       = 5
+	rpmTypeString      = 6
+	rpmTypeBin         = 7
+	rpmTypeStringArray = 8
+	rpmTypeI18nstring  = 9
+
+	signatureTagSize        = 1000
+	signatureTagMD5         = 1004
+	signatureTagPayloadSize = 1007
+	signatureTagSHA1        = 269
+
+	tagName              = 1000
+	tagVersion           = 1001
+	tagRelease           = 1002
+	tagEpoch             = 1003
+	tagSummary           = 1004
+	tagDescription       = 1005
+	tagBuildtime         = 1006
+	tagSize              = 1009
+	tagOs                = 1021
+	tagArch              = 1022
+	tagLicense           = 1014
+	tagGroup             = 1016
+	tagPayloadFormat     = 1124
+	tagPayloadCompressor = 1125
+	tagPayloadFlags      = 1126
+	tagPrein             = 1023
+	tagPreinProg         = 1085
+	tagDirNames          = 1118
+	tagBaseNames         = 1117
+	tagDirIndexes        = 1116
+	tagFileUsernames     = 1039
+	tagFileGroupnames    = 1040
+	tagFileSizes         = 1028
+	tagFileModes         = 1030
+	tagFileInodes        = 1096
+	tagFileDevices       = 1095
+	tagRpmVersion        = 1064
+	tagFileMtimes        = 1034
+	tagFileFlags         = 1037
+	tagFileLangs         = 1097
+	tagFileRdevs         = 1033
+	tagFileDigests       = 1035
+	tagFileLinkTos       = 1036
+	tagRequireFlags      = 1048
+	tagRequireName       = 1049
+	tagRequireVersion    = 1050
+	tagPayloadDigest     = 5092
+	tagPayloadDigestAlgo = 5093
+
+	rpmSenseLess         = 0x02
+	rpmSenseGreater      = 0x04
+	rpmSenseEqual        = 0x08
+	rpmSensePrereq       = 0x40
+	rpmSenseInterp       = 0x100
+	rpmSenseScriptPre    = 0x200
+	rpmSenseScriptPost   = 0x400
+	rpmSenseScriptPreun  = 0x800
+	rpmSenseScriptPostun = 0x1000
+)
+
+var (
+	headerMagic   = []byte{0x8e, 0xad, 0xe8}
+	versionMagic  = 0x01
+	reservedMagic = 0
+
+	knownFiles = map[string]struct{}{
```

Comment:
Will it work for Ubuntu, Centos, AltLinux,...? It should be carefully tested for the different platforms.

Also the name "knownFiles" tells me nothing

Reply:
Yes, it already works for all this systems. A part of this dirs is created by prein script (`"usr/share/tarantool` etc.).

I agree that naming can be better, what about `systemDirs` or `existingDirs`?

Reply:
`systemDirs `